scapy/tools/UTscapy.py

```diff
@@ -850,4 +850,4 @@ def main(argv):
     return glob_result
 
 if __name__ == "__main__":
-    exit(main(sys.argv[1:]))
+    sys.exit(main(sys.argv[1:]))
```

Comment:
Why do you need that? (real question)

Reply:
Two reasons:
- while creating the PR, I noticed than when a thread would bug and never end, UTscapy would never close and stand by forever. Not safe for CI tests.
- quoting the doc:
https://docs.python.org/3/library/constants.html#exit
> The site module (which is imported automatically during startup, except if the -S command-line option is given) adds several constants to the built-in namespace. They are useful for the interactive interpreter shell and should not be used in programs.